ruby_on_rails/forms_and_authentication/form_basics.md

```diff
@@ -97,7 +97,7 @@ Those inputs will now get transformed into a nested hash under the `:user` key.
 
 Specific parameters of the `params` hash are accessed like any other nested hash `params[:user][:email]`.
 
-Don't forget that you have to whitelist the params now in your controller using `require` and `permit` because they are a hash instead of just a flat string.  See the Controller section below for a refresher on the controller side of things.
+Don't forget that you have to allowlist the params now in your controller using `require` and `permit` because they are a hash instead of just a flat string.  See the Controller section below for a refresher on the controller side of things.
```

Comment:
```suggestion
Don't forget that you have to allow the params now in your controller using `require` and `permit` because they are a hash instead of just a flat string.  See the Controller section below for a refresher on the controller side of things.
```